slack.go

```diff
@@ -283,8 +283,8 @@ func (b *Bot) Start(manager JobManager) error {
 		},
 	})
 
-	slack.Command("build <from>", &slacker.CommandDefinition{
-		Description: "Create a new release image from one or more pull requests. The successful build location will be sent to you when it completes and then preserved for 12 hours.",
+	slack.Command("build <pullrequest>", &slacker.CommandDefinition{
```

Comment:
You cant change the name of the request parameter without also updating the logic that parses it out (Line 296).

Reply:
thanks, fixed